config.py

```diff
@@ -0,0 +1,8 @@
+import os
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+# Attributes of the database:
+CSRF_ENABLED = True
+SECRET_KEY = 'A0Zr98j/3yX R~XHH!jmN]LWX/,?RT'
```

Comment:
"SECRET" and "on github" don't go together well. This needs to *not* be in version control.